### DISCUSSION TITLE
Installing firedrake with conda

### DISCUSSION THREAD

**Discussion by martintruffer** [firedrakeproject/firedrake#3640] [Install FAQ]
The installation page makes it sound very complicated to install firedrake if conda is already installed on the system. But the following steps worked really nicely for a number of people in our lab (running Linux):
1) 'deactivate conda'
2) modify the PATH variable by deleting the anaconda entries that are put at the very beginning of the path
3) install firedrake

After that, all that needs to happen in the shell is to 'deactivate conda' and 'source firedrake/bin/activate'.

**Comment by Ig-dolci** [comment]
Thank for your report! I just added your advices [here](https://github.com/firedrakeproject/firedrake/wiki/Install-Frequently-Asked-Questions/_edit).
